aiohttp_devtools/runserver/config.py

```diff
@@ -83,15 +89,39 @@ def __init__(self, *,
             self.host = bind_address
 
         self.bind_address = bind_address
+        if main_port is None:
+            main_port = DEFAULT_PORT if ssl_context_factory_name is None else DEFAULT_PORT + 443
         self.main_port = main_port
-        self.aux_port = aux_port or (main_port + 1)
+        if aux_port is None:
+            aux_port = main_port + 1 if ssl_context_factory_name is None else DEFAULT_PORT + 1
+        self.aux_port = aux_port
         self.browser_cache = browser_cache
+        self.ssl_context_factory_name = ssl_context_factory_name
+        self.ssl_rootcert_file_path = ssl_rootcert_file_path
         logger.debug('config loaded:\n%s', self)
 
+    @property
+    def protocol(self) -> Literal["http", "https"]:
+        return "http" if self.ssl_context_factory_name is None else "https"
+
     @property
     def static_path_str(self) -> Optional[str]:
         return str(self.static_path) if self.static_path else None
 
+    @property
+    def client_ssl_context(self) -> Union[SSLContext, None]:
+        client_ssl_context = None
+        if self.protocol == 'https':
+            client_ssl_context = create_default_ssl_context()
+            if self.ssl_rootcert_file_path:
+                try:
+                    client_ssl_context.load_verify_locations(self.ssl_rootcert_file_path)
+                except FileNotFoundError as e:
+                    raise AdevConfigError('{}: {}'.format(e.strerror, self.ssl_rootcert_file_path))
+                except Exception:
+                    raise AdevConfigError('invalid root cert file: {}'.format(self.ssl_rootcert_file_path))
```

Comment:
Is it difficult to create a test for this code? Seems to be the only missing coverage now.